src/ResourceManagement/Sql/SqlManagement/Generated/DatabaseOperations.cs

```diff
@@ -772,6 +772,10 @@ public SqlManagementClient Client
                     {
                         result.Status = OperationStatus.Succeeded;
                     }
+                    if (statusCode == HttpStatusCode.OK)
```

Comment:
i would suggest you get rid of the whole generated folder under test folder. I just excluded them from the project, but those file are pretty much duplicates. Should go